backend/src/main/java/com/empire/World.java

```diff
@@ -2065,13 +2088,10 @@ void evaluateGothiSpells() {
 					notifications.add(new Notification(k, "Earthquakes", notification));
 				}
 			}
-			if (tivar.veil) {
-				for (Region r : regions) r.crops *= .97;
-			}
-			if (tivar.deluge) {
-				for (Region r : regions) r.crops *= .97;
-				
-			}
+			damageCrops(tivar.deluge);
```

Comment:
I know this came up previously, I don't recall if this was the same context.  But damaging the crops this way doesn't additively stack the percentages when multiple Gothi spells or potential other crop-damaging effects happen in a region.  That in itself isn't a huge issue, but with the new progressive damage scale the order in which the spells' effects are applied also makes a difference depending on how longer-running spells are earlier/later in the order.

Reply:
That's correct, they're multiplicative. I can clarify that they are independent in the rules doc.

As for the order mattering, I don't think that's true, by the commutative property of multiplication, e.g. the net result here is 
crops = crops * (1-damage1) * (1-damage2) * (1-damage3) * (1-damage4)
and that should be the same regardless of what order they are applied in. Right?

Reply:
You're right, not sure what I was thinking at the time, I had just gotten back from running.